target-app/Home/Models/GetTargetResponse.swift

```diff
@@ -9,6 +9,10 @@ import Foundation
 
 struct GetTargetResponse: Codable {
     let targets: [TargetElement]
+    
+    static func toDomain(_ list: [TargetElement]) -> [Target] {
+        return list.map { $0.target as Target }
```

Comment:
since `TargetElement.target` is already of type `Target` do you really need `as Target` ?

Reply:
Why is this a static function since you need the target element list?
Maybe an instance function makes more sense, so you can simply do `targetResponse.toDomain()`